### ISSUE TITLE
Missing package

### ISSUE THREAD

**Issue by adamiusz** [katspaugh/wavesurfer.js#2097]
I ue wordpress and wavesurfer.js and everything was OK until yesterday.

Yesterday I noticed that the plugin is not working properly and I have a warning in the console:

`DevTools failed to load SourceMap: Could not load content for https://unpkg.com/wavesurfer.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE`

and when I enter url `https://unpkg.com/wavesurfer.js.map`

I have error in notice:

`Cannot find package wavesurfer.js.map@latest`

How fix this problem?

**Comment by sundayz** [comment]
This is because you are using the URL which points to the latest release of wavesurfer. You should always use an unpkg url with a version included, for example here is 4.2.0: https://unpkg.com/wavesurfer.js@4.2.0/dist/wavesurfer.js

By including a version number in the URL, your site will never break if there is a broken release, or if there is a missing release (the current version has probably not been uploaded yet)

**Comment by thijstriemstra** [comment]
>  You should always use an unpkg url with a version included, for example here is 4.2.0: https://unpkg.com/wavesurfer.js@4.2.0/dist/wavesurfer.js

Agreed. But don't use unpkg in production, host the wavesurfer.js script yourself.

**Comment by adamiusz** [comment]
So what i must insert here:

`wp_register_script('wavesurfer_js', 'https://unpkg.com/wavesurfer.js');
wp_enqueue_script('wavesurfer_js', 'https://unpkg.com/wavesurfer.js'); `

**Comment by thijstriemstra** [comment]
> update the package soon to be as before?

What package?

**Comment by adamiusz** [comment]
What I must insert  above not to receive this message?

**Comment by sundayz** [comment]
https://unpkg.com/wavesurfer.js@4.2.0/dist/wavesurfer.js

try that

**Comment by adamiusz** [comment]
still have in console:

_`DevTools failed to load SourceMap: Could not load content for https://unpkg.com/wavesurfer.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE`_
